### DISCUSSION TITLE
Spacebar is toggling inside input as opposed to typing the spacebar

### DISCUSSION THREAD

**Discussion by WildEgo** [floating-ui/floating-ui#1725] [Q&A] [answered]
Hey guys, it's just as the title describes.
Here's me replicating it in [CodeSandbox](https://codesandbox.io/s/ancient-silence-gzcmy4?file=/src/App.tsx)

Is there a way to disable that and just make it a true typeahead?

Regards,
Luis Bizarro

**Comment by atomiks** [comment, marked as answer]
`useClick()` will add keyboard handlers that enable Spacebar to toggle the open state. Maybe you want to `useFocus()` instead?

Here's `useFocus()`: https://codesandbox.io/s/wild-wave-7iuo2e?file=/src/App.tsx (note `keyboardOnly` needs to be `false`). This makes more sense than `useClick`, imo. Further `useTypeahead` stops Spacebar from propagating, you probably should just use a custom filter?

**Reply by WildEgo** [reply]
Thank you that indeed worked, useFocus seems to do the trick as opposed to the other two! Sorry for the inconvenience.
